### ISSUE TITLE
Support Windows High Contrast Mode

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#926]
**Describe the bug**
The Fiori 3 design specification supports a High Contrast Black and High Contrast White theme. See https://wiki.wdf.sap.corp/wiki/pages/viewpage.action?pageId=2144577961.

However, when a Windows High Contrast Mode user opens any SAP site or app on the web, the underlying SAP theme (high contrast or otherwise) will not matter as it (i.e. the styles) would be replaced by the native high contrast settings with a limited color palette configure by the user at the OS level (see [wiki](https://github.com/SAP/fundamental-styles/wiki/Windows-High-Contrast-Mode-Exploration#windows-high-contrast-mode)). Hence, if we plan to support this mode, we have to make sure that all the SAP UI5 designs (high contrast or otherwise) comply with the minimum required accessibility standards and follow the best practices for HCM, proposed [here](https://github.com/SAP/fundamental-styles/wiki/Windows-High-Contrast-Mode-Exploration#proposed-criteria-and-accessibility-best-practices). We should plan to support this mode since expecting the user to disable native Windows HCM when using the SAP products and using the SAP Themes instead is not optimal. If we do not want to support Windows HCM, we may also figure out if there is a way to

1. Know if Windows HCM is enabled in the browsers
2. Skip the Windows HCM for SAP website/web app pages
3. Apply Fiori 3 HCM themes instead.

>What version of the Fundamental Library Styles are you using?
v0.8.1

**To Reproduce**
See [this wiki](https://github.com/SAP/fundamental-styles/wiki/Windows-High-Contrast-Mode-Exploration) for detailed steps to reproduce and findings.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: IE
 - Version [e.g. 11]
